packages/Panel/README.md

```diff
@@ -31,7 +31,7 @@ npm install @paprika/panel
 | height                | [string,number]                                                                             | false    | "33%"                         | The height of the open Panel (when slide in from bottom)                                                                                        |
 | isCompact             | bool                                                                                        | false    | false                         | Control the compactness of the Panel                                                                                                            |
 | isInline              | bool                                                                                        | false    | false                         | Render the panel inline                                                                                                                         |
-| isOpen                | bool                                                                                        | true     | -                             | Control the visibility of the Panel. This prop makes the Panel appear.                                                                          |
```

Comment:
could this be a breaking change? if it used to be true by default?

Reply:
It was `required` `true`, not default value `true`. I changed it to `optional` with the default value `false`. Because if you don't pass in the value, `undefined` will act the same as `false` anyway, but more problematic 
What do you think?